### ISSUE TITLE
CServerExoAppInternal__SetPauseState missing in .35 API

### ISSUE THREAD

**Issue by Shad000w** [nwnxee/unified#1624]
.35 still allows to save game, but I no longer see this function anywhere. Is it a bug in API or NWN-EE changed the way how savegame works?

**Comment by Daztek** [comment]
SetPauseState is here: https://github.com/nwnxee/unified/blob/master/NWNXLib/API/API/CServerExoAppInternal.hpp#L228

If you mean CServerExoAppInternal::SaveGame, the mangled name is in FunctionsLinux.hpp, I don't know why the func decl isn't in the api

**Comment by Shad000w** [comment]
Yeah I meant SaveGame, for some reason I copypasted function name of the hook above SaveGame.
